Libraries/LibCrypto/Cipher/Mode/GCM.h

```diff
@@ -94,20 +119,14 @@ class GCM : public CTR<T, IncrementFunction> {
 
         auto auth_tag = m_ghash->process(aad, out);
         block0.apply_initialization_vector({ auth_tag.data, array_size(auth_tag.data) });
-        block0.bytes().copy_to(tag);
+        tag.overwrite(0, block0.bytes().data(), tag.size());
```

Comment:
these two should be equivalent, no?

Reply:
Maybe that API is called `copy_trimmed_to`? I seem to remember something like that.

Reply:
`copy_trimmed_to` does the job!